frontend/src/components/Admin/AddUser/AddUser.tsx

```diff
@@ -74,6 +75,8 @@ export default function AddUser(): JSX.Element {
     currentSelectedMember: allMembers[0],
     isCreatingUser: false
   });
+  const [csvFile, setCsvFile] = useState<File | undefined>(undefined);
+  const [uploadStatus, setUploadStatus] = useState<string>('');
```

Comment:
More like, `uploadError`, at least based on how it's being used.

Reply:
It's also used to display a success message indicating how many members were uploaded, so `uploadError` is a little imprecise.
<img width="300" alt="image" src="https://github.com/cornell-dti/idol/assets/69322572/44b99de6-1b54-4f45-b506-17249acaf5d5">

I've changed the hook to include a status and a message which is rendered as either a green success message or a red error message.
<img width="303" alt="image" src="https://github.com/cornell-dti/idol/assets/69322572/0682d5dd-eb14-4cac-a9a2-8686cbaf25d7">
<img width="303" alt="image" src="https://github.com/cornell-dti/idol/assets/69322572/7f4f10f7-7ac2-4f95-bb77-f911daef6888">

Reply:
I agree. I didn't see that you had the `Uploaded` message originally. I like what you have now.